### PR TITLE
update quic data bridge

### PR DESCRIPTION
**Pull request by LenaLenaPan**

### CODE REVIEW

**Review by lee-emqx — CHANGES_REQUESTED** [review]
Pls remove nng from .gitignore

File: docs/zh_CN/bridges/quic-bridge.md [hunk 1]
```diff
@@ -83,88 +84,177 @@ bridges.mqtt.name {
...
-   forward1/msg: forward_msg
+   $ ./nanomq_cli sub --quic -h "your.host.address"  -t "forward1/#" -q 2 -u emqx -P emqx123
    ```
 
-2. 发布消息到本地**NanoMQ** Broker 主题:
+2. 新建一个命令行窗口，发布消息到 **NanoMQ** Broker，主题为 “`forward1/msg`”：
 
-   发布消息到**NanoMQ** Broker ，主题为 “`forward1/msg`”：
+   ```bash
+   $ ./nanomq_cli pub --quic -h "your.host.address"  -t "nanomq/1" -m "forward_msg" -q 2
```
Comment (lee-emqx) [inline docs/zh_CN/bridges/quic-bridge.md:168]: ./nanomq_cli pub --quic -h "your.host.address" -t "forward1/msg" -m "forward_msg" -q 2

Comment (lee-emqx) [reply docs/zh_CN/bridges/quic-bridge.md:168]: 这里两个 your.host.address 最好区分一下

Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:168]:
```suggestion
   $ ./nanomq_cli pub --quic -h "your.host.address"  -t "forward1/msg" -m "forward_msg" -q 2
```

File: docs/zh_CN/bridges/quic-bridge.md [hunk 2]
```diff
@@ -83,88 +84,177 @@ bridges.mqtt.name {
...
+### 测试消息接收
 
-1. 订阅本地 NanoMQ Broker 的主题：
+1. 为本地 NanoMQ Broker 订阅消息主题：
 
-   从**NanoMQ**订阅主题 “`cmd/topic1`”, 该主题将接收到**EMQX**上发布的数据：
+   为 **NanoMQ** 订阅主题 “`cmd/topic1`”，用于接收 **EMQX** 发布的数据：
 
-   在第 3 个终端上执行订阅:
+   新建第三个命令行窗口，前往 build 文件夹下的 nanomq_cli 文件夹，执行以下命令进行订阅：
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:183]:
```suggestion
   在第二个命令行窗口中，前往 build 文件夹下的 nanomq_cli 文件夹，执行以下命令进行订阅：
```

File: docs/zh_CN/bridges/quic-bridge.md [hunk 3]
```diff
@@ -83,88 +84,177 @@ bridges.mqtt.name {
...
 
    ```bash
-   $ nanomq_cli sub -t "recv/topic1"
-   recv/topic1: cmd_msg
+   $ ./nanomq_cli sub --quic -h "your.host.address"  -t "recv/topic1" -q 2
    ```
+   
+2. 新建第四个命令行窗口，发布消息到远端 **EMQX** Broker，主题为 “`cmd/topic1`”：
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:189]:
```suggestion
2. 在第一个命令行窗口，发布消息到远端 **EMQX** Broker，主题为 “`cmd/topic1`”：
```

File: docs/zh_CN/bridges/quic-bridge.md [hunk 4]
```diff
@@ -83,88 +84,177 @@ bridges.mqtt.name {
...
 
-2. 发布消息到远端**EMQX** Broker 主题“`cmd/topic1`”：
-
-   在第 4 个终端上执行消息发布:
+   ```bash
+   $ ./nanomq_cli pub --quic -h "your.host.address"  -t "recv/topic1" -m "cmd_msg" -q 2 -u emqx -P emqx123
+   ```
+   
+3. 返回第三个命令行窗口，将能看到远端 **EMQX** Broker 发送的消息，例如：
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:195]:
```suggestion
3. 返回第二个命令行窗口，将能看到远端 **EMQX** Broker 发送的消息，例如：
```

**Review by JaylinYu — APPROVED** [review]

**Pull request merged**